python/python/res/server/simulation_context.py

```diff
@@ -175,3 +185,31 @@ def job_status(self, iens):
         except ValueError:
             return None
         return self._queue_manager.getJobStatus(queue_index)
+
+
+    def status_timestamp(self):
+        """Will return a timestamp for the last status change of the simulations.
+
+        The timestamp is related to status changes when a simulation has
+        started, completed and failed.
+
+        """
+        return self._queue_manager.status_timestamp()
+
+
+    def progress_timestamp(self, iens = None):
+        """
+        Will return a timestamp for when the simulation has progressed to a new forward model step.
+        """
+
+        if iens is None:
+            return self._queue_manager.progress_timestamp()
+        else:
```

Comment:
This `else` isn't necessary and just adds an additional level of indentation.  Please remove and de-indent the subsequent block.